ldif_test.go

```diff
@@ -0,0 +1,118 @@
+package ldap_test
+
+import (
+	"bytes"
+	"gopkg.in/ldap.v2"
+	"testing"
+)
+
+var ldifRFC2849Example = `version: 1
+dn: cn=Barbara Jensen, ou=Product Development, dc=airius, dc=com
+objectclass: top
+objectclass: person
+objectclass: organizationalPerson
+cn: Barbara Jensen
+cn: Barbara J Jensen
+cn: Babs Jensen
+sn: Jensen
+uid: bjensen
+telephonenumber: +1 408 555 1212
+description: A big sailing fan.
+
+dn: cn=Bjorn Jensen, ou=Accounting, dc=airius, dc=com
+objectclass: top
+objectclass: person
+objectclass: organizationalPerson
+cn: Bjorn Jensen
+sn: Jensen
+telephonenumber: +1 408 555 1212
+`
```

Comment:
if I'm reading the rfc correctly, any number of spaces (including 0) are allowed between the ":" and the attribute value. would be good to add a test entry that omits the space, and one that has multiple spaces (and validate the resulting entries)